FParsecCS/Text.cs

```diff
@@ -395,11 +392,11 @@ public static int CountTextElements(string str)
 
 #else
 
-internal unsafe struct IsWhitespaceHelper {
+internal unsafe static class IsWhitespaceHelper {
```

Comment:
How about I remove this entire custom `IsWhitespace` implementation? .NET does something similar.

https://github.com/dotnet/runtime/blob/ec26662ac9f1e89b2556d81fa311f91ab3abd492/src/libraries/System.Private.CoreLib/src/System/Globalization/CharUnicodeInfo.cs#L182-L191

Reply:
Yeah, the performance is probably similar nowadays, though it may be worth double checking.

Reply:
😳

|  Method |     Mean |   Error |  StdDev |
|-------- |---------:|--------:|--------:|
|  DotNet | 325.0 μs | 3.88 μs | 3.63 μs |
| FParsec | 225.8 μs | 4.26 μs | 3.78 μs |

```csharp
using BenchmarkDotNet.Attributes;
using BenchmarkDotNet.Running;
using FParsec;

BenchmarkRunner.Run<IsWhitespaceBenchmark>(args: args);

public class IsWhitespaceBenchmark
{
    [Benchmark]
    public int DotNet()
    {
        int count = 0;
        for (char c = '\0'; c != char.MaxValue; c++)
            if (char.IsWhiteSpace(c))
                count++;
        return count;
    }
    
    [Benchmark]
    public int FParsec()
    {
        int count = 0;
        for (char c = '\0'; c != char.MaxValue; c++)
            if (Text.IsWhitespace(c))
                count++;
        return count;
    }
}
```

Reply:
Thanks for measuring! Then maybe let's keep the custom implementation until it becomes a significant hassle to maintain.